go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/meroxa/cli
 
 go 1.20
 
+replace github.com/meroxa/meroxa-go => ../meroxa-go
```

Comment:
Just a reminder to update this once https://github.com/meroxa/meroxa-go/pull/189/files ships.